spec/rails_helper.rb

```diff
@@ -8,8 +8,11 @@
 require 'shoulda-matchers'
 require File.join(File.dirname(__FILE__), 'support/valid_attribute')
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
+require File.join(File.dirname(__FILE__), 'support/database_cleaner')
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.